CMakeLists.txt

```diff
@@ -70,7 +86,23 @@ add_subdirectory(${CMAKE_CURRENT_LIST_DIR}/src/plasma/)
 add_subdirectory(${CMAKE_CURRENT_LIST_DIR}/src/local_scheduler/)
 add_subdirectory(${CMAKE_CURRENT_LIST_DIR}/src/global_scheduler/)
 
+# final target copy_ray
+add_custom_target(copy_ray ALL)
+
+# Make sure redis-server is ready before copying.
+add_dependencies(copy_ray copy_redis)
+add_dependencies(copy_ray ray_redis_module)
+
+# copy plasma_store_server
+add_custom_command(TARGET copy_ray POST_BUILD
+  COMMAND ${CMAKE_COMMAND} -E
+    copy ${ARROW_HOME}/bin/plasma_store_server ${CMAKE_CURRENT_BINARY_DIR}/src/plasma
+  )
```

Comment:
I'd slightly prefer to not have the ) on a new line, seems more common in cmake

Reply:
OK, I just have no idea about the cmake language format. I will change all such cases.